TestProject/API/APIClient.swift

```diff
@@ -18,7 +18,7 @@ class APIClient {
         }
     }
 
-    static func getInformation(completion:@escaping (AFResult< [SampleModel]>)->Void) {
+    static func getInformation(completion:@escaping (AFResult< [Data]>)->Void) {
```

Comment:
Let's practice having a naming convention for our model classes. For eg., `UserModel`, `BillHistoryModel`, so that it would be more descriptive.